sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/lang/IntermediateType.kt

```diff
@@ -126,7 +139,9 @@ internal data class IntermediateType(
     }
 
     column?.adapter()?.let { adapter ->
-      val adapterName = (column.parent as Queryable).tableExposed().adapterName
+      // In the OG test, column.parent is SqlCreateTableStmtImpl(CREATE_TABLE_STATEMENT).
+      // In the new test, column.parent is MODULE_ARGUMENT_DEF
```

Comment:
Namely, in the case of a virtual table, the column hiearchy looks like this:
```
Column (COLUMN_DEF) -> parent (MODULE_ARGUMENT_DEF) -> parent (MODULE_ARGUMENT) -> parent (SqlCreateVirtualTableStmtImpl)
```

vs
```
Column (COLUMN_DEF) -> parent (SqlCreateTableStmtImpl)```
in the case of a non-fts/concrete table.